azDevOps/azure/templates/steps/publish-nuget.yml

```diff
@@ -0,0 +1,11 @@
+parameters:
+  nugetFeedName: "Next-Ecommerce"
+
+steps:
+- task: DotNetCoreCLI@2
+  displayName : Nuget Push
+  inputs:
+    command: 'push'
+    packagesToPush: '$(Build.ArtifactStagingDirectory)/*.nupkg'
+    nuGetFeedType: 'internal'
+    publishVstsFeed: ${{ parameters.nugetFeedName }}
```

Comment:
editorconfig should take care of this - please include a new line at the end of yaml files